table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -123,13 +122,8 @@ public DatabaseInfo getDatabaseInfo() throws IOException {
           ? "" : glueDatabase.getLocationUri();
       String glueDbDescription = glueDatabase.getDescription() == null
           ? "" : glueDatabase.getDescription();
-      Map<String, String> glueParameters = new HashMap<>();
-      Map<String, String> parameters = glueDatabase.getParameters();
-      if (parameters != null) {
```

Comment:
I feel like the previous code was better, since we still create a new map for our own internal data structure. What does the new code fix?

We could change this code a little bit to be something like:
```
if (parameters != null) {
  glueParameters.putAll(parameters);
}
```

Reply:
yeah, it's better, changed.